slider/position.py

```diff
@@ -16,3 +16,21 @@ class Position(namedtuple('Position', 'x y')):
     The visible region of the osu! standard playfield is [0, 512] by [0, 384].
     Positions may fall outside of this range for slider curve control points.
     """
+
+class Point(namedtuple('Point', 'x y offset')):
```

Comment:
Yeah I only really used `Point` because I'd used that in some code I wrote before this and wanted to easily copy across stuff. Feel free to change it though I don't have any good suggestions. Maybe using `Point`s for everything and just leaving the offset as None if there isn't a time? I'll leave it to your judgement, since it is your library :P